lib/renaissance/helpers/adapt.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Renaissance.Helpers.Adapt do
```

Comment:
The `Adapt` name is a little bit unclear. Maybe just `Renaissance.Format.Money`? Everything in here has to do with Money.